sdk/kms/tests/e2e-aws-kms.test.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { beforeAll, describe, expect, it } from 'vitest';
+
+import { AwsKmsSigner } from '../src/aws/aws-kms-signer';
+
+describe('Aws KMS signer E2E testing', () => {
```

Comment:
I wonder if we could have these tests skipped if you're missing the AWS config and `process.env.CI !== 'true'` just so you can still `pnpm turbo test` and have it work?

Reply:
Or even more ideally, mock the AWS responses and just assert that it makes the expected HTTP requests so that these tests have no external dependencies (I would love you forever if you did that)

Reply:
I like having an E2E test that tests _everything_, and also see the value in mocking out the external calls...

Let me follow up in the future with a PR to mock out the AWS responses (and acquire your love forever ❤️ )